packages/gallery/src/components/gallery/layoutingIndex.js

```diff
@@ -45,9 +45,24 @@ export default class BaseGallery extends React.Component {
       GalleryComponent = LeanGallery;
     } else {
       if (!this.state.blueprint || Object.keys(this.state.blueprint).length === 0){
-        return null;
+        try {
+          const {layoutFixer} = window;
+          if (typeof window !== 'undefined' && layoutFixer && !layoutFixer.hydrated) {
```

Comment:
why not use this as the initial state in the constructor (if there is a layoutFixer, state.blueprints... = layoutFixer...
Otherwise you are in a race wit the onNewProps in the constructor. This should replace it so an initial state wont be set with the first props that arrive.

constructor {
if (layoutFixer) {
state.blueprint = fixer 
} else {
onNewProps(props)
}
}

This will also not put layoutFixer logic in the "canRender".

If you exit the constructor w/o a layoutFixer blueprint you will return null until the state shows a valid blueprint.